### PR TITLE
Migrate 138_IPI on GCP filestore CSI profile to Prow

### PR DESCRIPTION
**Pull request by chao007**
Main function:
Add workflow `cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi` which is install gcp filestore csi operator and driver, create storageclass

### CODE REVIEW

**Comment by chao007** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-gcp-ipi-compact-filestore-csi-p2-f14

**Comment by chao007** [comment]
/pj-rehearse pull-ci-openshift-gcp-filestore-csi-driver-operator-release-4.14-operator-e2e

**Comment by chao007** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-gcp-ipi-compact-filestore-csi-p2-f14

**Comment by chao007** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-gcp-ipi-compact-filestore-csi-p2-f14

**Comment by chao007** [comment]
Hi @jiajliu @gpei @jianli-wei , I would like to migrate GCP Filestore profile to Prow, can you help to review when have time?
Thanks.

**Comment by chao007** [comment]
@liangxia Could you help to review this pr? Thanks.

File: ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.metadata.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	"path": "cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.yaml",
```
Comment (jianlinliu) [inline ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.metadata.json:2]: For compact cluster, we are setting `SIZE_VARIANT: "compact"` ENV in job configuration level, so for the workflow, you are supposed to create a non-compact workflow here, that means, the path should be like `gcp/ipi/filestore-csi/`

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.metadata.json:2]: so you decide not combine the filestore-csi to the compact install workflow?

Comment (chao007) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.metadata.json:2]: If set `SIZE_VARIANT: "compact"` workers will be 0 right? I prefer to
create master + worker env https://github.com/openshift/release/blob/1ed184c3693b8e2680ca0c6e0e94cd0e34d8fd23/ci-operator/step-registry/ipi/conf/gcp/ipi-conf-gcp-commands.sh#L16

Comment (jianlinliu) [reply ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/compact/filestore-csi/cucushift-installer-rehearse-gcp-ipi-compact-filestore-csi-workflow.metadata.json:2]: > If set SIZE_VARIANT: "compact" workers will be 0 right? 

The cluster would only have 3 nodes, each node take both master and worker roles. I thought you select compact install workflow to combine filestore-csi on purpose. If you prefer a more common cluster, you can select other install workflow to combine your feature, e.g: gcp-ipi-ovn-ipsec-p1-f4

**Comment by chao007** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.13-amd64-nightly-gcp-ipi-compact-filestore-csi-p2-f14

**Comment by chao007** [comment]
@bertinatto @dobsonj Hi can you help to review this pr?
I update the scripts of [storage-destroy-csi-gcp-filestore-commands.sh](https://github.com/openshift/release/blob/6cd71493202f49ce391d1e83cd824fccd0234c07/ci-operator/step-registry/storage/destroy/csi-gcp-filestore/storage-destroy-csi-gcp-filestore-commands.sh) and add chain ci-operator/step-registry/storage/conf/csi-optional/gcp-filestore/storage-conf-csi-optional-gcp-filestore-chain.yaml for QE job

**Pull request merged**